udata/core/dataset/models.py

```diff
@@ -582,8 +582,17 @@ def check_availability(self):
 
     @property
     def last_update(self):
+        """
+        Use the more recent date we would have on resources (harvest, published, modified).
```

Comment:
Trying out custom logic for `last_update`, described in the api fields as `The resources last modification date`.